### ISSUE TITLE
[Suggestion] Small improvement for chat interface: Enter to send

### ISSUE THREAD

**Issue by ramarivera** [brianpetro/obsidian-smart-connections#444]
I think it would be a lot more fluid if we could send our messages in the chat interface with Enter, or Cmd/Ctrl + Enter maybe, like some messaging apps.

Let me know if this was already planned, and feel free to close it if so :)

**Comment by brianpetro** [comment]
This is a pretty common request.

I remember there were some complications, and that's why I didn't implement something like this sooner. The other issue is that people have different settings that they're familiar with, so it would probably be best if this was customizable on some level.

I'll be looking into it as I'm working on v2.1 where I'll be re-writing the Smart Chat feature.

🌴

**Comment by RealRavens** [comment]
Agreed, it's more common, but ya the option to customize it is probably best. I probably wouldn't recommend just enter, only because of how creative some prompts can be.

**Comment by GoulartNogueira** [comment]
That must be made very carefully.
In some applications, I often try to break lines with ctrl+Enter, or shift+Enter.
And frequently my incomplete message is accidentally sent.

In this case, it's worse because it can ruin the conversation context and GPT calls costs money.
